http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -169,56 +165,47 @@ class ProxyService(
    */
   def proxyRequest(userInfo: UserInfo, googleProject: GoogleProject, clusterName: ClusterName, request: HttpRequest)(
     implicit ev: ApplicativeAsk[IO, TraceId]
-  ): Future[HttpResponse] =
+  ): IO[HttpResponse] =
     authCheck(userInfo, googleProject, clusterName, ConnectToCluster).flatMap { _ =>
       proxyInternal(userInfo, googleProject, clusterName, request)
     }
 
+  def getTargetHost(googleProject: GoogleProject, clusterName: ClusterName): IO[HostStatus] =
+    Proxy.getTargetHost[IO](clusterDnsCache, googleProject, clusterName)
+
   private def proxyInternal(userInfo: UserInfo,
                             googleProject: GoogleProject,
                             clusterName: ClusterName,
-                            request: HttpRequest): Future[HttpResponse] = {
-    logger.debug(s"Received proxy request for user $userInfo")
+                            request: HttpRequest): IO[HttpResponse] = {
+    logger.info(s"Received proxy request for user $userInfo: ${clusterDnsCache.stats} / ${clusterDnsCache.size}")
```

Comment:
I think we should leave this at `debug` -- proxy requests can be pretty high volume.

Reply:
yea, this was when I was debugging the weird path issue....will revert